lib/taiko.js

```diff
@@ -3338,9 +3338,12 @@ const evaluate = async (selector, func) => {
 };
 
 const validate = () => {
-  if (!dom || !page) {
+  if (!_client) {
     throw new Error('Browser or page not initialized. Call `openBrowser()` before using this API');
   }
+  if (_client._ws.readyState > 1) {
+    throw new Error('Connection to Browser closed. Try reconnecting by `openBrowser()`');
```

Comment:
If browser instance is already running, `openBrowser` will error out stating only one instance is allowed. If there isn't a browser instance, it will launch a new one, but we cannot consider that as 'reconnecting'

Reply:
Updated the messaging to reflect if the browser instance is closed.